Client/src/styles/typography.css

```diff
@@ -1,4 +1,13 @@
 body {
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji',
     'Segoe UI Emoji', 'Segoe UI Symbol';
+    color: var(--black);
+}
+
+a, a:visited {
```

Comment:
Ik zou deze file graag puur houden voor de typography. Misschien een `core.css` aanmaken waarin we dit soort styling van tags definieren?